server/middleware/populateCurrentUser.ts

```diff
@@ -8,6 +8,9 @@ export default function populateCurrentUser(userService: UserService): RequestHa
       if (res.locals.user) {
         const user = res.locals.user && (await userService.getUser(res.locals.user.token))
         if (user) {
+          if (!user.activeCaseLoadId) {
```

Comment:
In practice this shouldn't get called: although undocumented, `getUser` (from manage users API) does in fact return case load. Looking up separately though as advised not to rely on this property being present.